index.html

```diff
@@ -0,0 +1,160 @@
+            </li>
+        </ul>
+    </div>
+    <main>
+        <section id="introduction">
+            <h3 class="introduction-greet">"Hello! Data Scientists"</h3>
+            <h1 class="introduction-h1">AI & BIG DATA FOR A SMARTER FUTURE EXHIBITION 2023</h1>
+            <P class="introduction-para">
+                The world is becoming more hyper-connect with data every day.
+                Data literacy is so critical to our education system but also
+                key to humanity. This Exhibition will showcase the next generation
+                enterprise technologies and strategies from the world of Data Science
+                and AI, providing an opportunity to explore and discover the practical
+                and successful implementation of data science and machine learning to 
+                drive your business forward.
+            </P>
+            <div class="introduction-div">
+                <h2 class="intro-div-h2">2023.01.19(THUR) ~ 20(FRI)</h2>
+                <p class="intro-div-para">@ Riverside Drive, Nairobi, Kenya.</p>
+            </div>
+        </section>
+        <section id="program-section">
+            <h2>Main Program</h2>
+            <hr>
+            <div id="programs">
+                <div class="program">
+                    <img src="images/lecture.png" alt="Lecture">
+                    <h4>Lecture</h4>
+                    <p>Listen to the speakers from various countries about securing value through AI & Big Data.</p>
+                </div>
+                <div class="program">
+                    <img src="images/exhibition.png" alt="Exhibition">
+                    <h4>Exhibition</h4>
+                    <p>Appreciate various creations applying successful implementation of data science and machine learning to drive your business forward.</p>
+                </div>
+                <div class="program">
+                    <img id="" src="images/forum.png" alt="Forum">
+                    <h4>Forum</h4>
+                    <p>Have time to share your thoughts and opinions with experts from each topic.</p>
+                </div>
+                <div class="program">
+                    <img src="images/workshop.png" alt="Workshop">
+                    <h4>Workshop</h4>
+                    <p>Try creating your own work using open source data science software rather than just watching.</p>
+                </div>
+                <div class="program">
+                    <img src="images/ignite.png" alt="Ignite">
+                    <h4>Ignite</h4>
+                    <p>Get opportunities to network with numerous clients, prospects, and partners from around the world.</p>
+                </div>
+            </div>
+            <button type="button" class="program-btn">Join AI & Big Data Expo 2023</button>
+            <p class="whole-program">
+                <a href="#">SEE THE WHOLE PROGRAM</a>
+            </p>
+        </section>
```

Comment:
- [ ] Please make sure that your program section is similar to the Figma template . I think you could reduce the height, font weight, and font size and also make sure that all the par for every single program card part is exactly the size as defined in the original design. you can check the screenshots below to see the differences. Please fix it in order to stick the design as much as possible :+1: 

#### Original design
![Screenshot from 2022-12-02 17-58-34](https://user-images.githubusercontent.com/6615394/205356829-aa1c49fa-6989-45bb-9a21-171c786513f6.png)

#### Your design
![Screenshot from 2022-12-02 17-58-02](https://user-images.githubusercontent.com/6615394/205356834-cf25dfa8-2431-4bcd-85fc-3e6f59b43740.png)